Add modules.d info to metricbeat docs

dedemorton opened the pull request:
Closes #5207

exekias commented on metricbeat/docs/metricbeat-options.asciidoc:
```diff
@@ -1,30 +1,85 @@
 
+// REVIEWERS: We aren't very consistent about when to show enabled: true. I'm
+// assuming this option is available for all modules, but wondering if we should
+// show it in the config.
```
We had some discussion about `enabled: true`, the idea is that having that it's the default and we provide an alternative to manage enabled/disabled status (renaming the yml file), it looks like a good idea to hide that so we don't provide to ways of doing the same thing

dedemorton replied:
Ok, I'll remove it from the examples then.

exekias commented on metricbeat/docs/metricbeat-options.asciidoc:
```diff
@@ -45,90 +100,107 @@ metricbeat.modules:
-      hosts: ["host1"]
-      period: 30s
-      enabled: true
+metricbeat.modules:
```
is this line still wanted? I'm not sure from the context

dedemorton replied:
yah, because this sections shows how to configure multiple modules in the metricbeat.yml file. I don't want users to get confused and add this to one of the config files under modules.d.

exekias commented on metricbeat/docs/metricbeat-options.asciidoc:
```diff
@@ -45,90 +100,107 @@ metricbeat.modules:
+the `set2` metricset will be fetched every 2 minutes:
+
+// REVIEWERS: I'm assuming the following config is still supported if you are
+// using the modules.d layout. Is that correct?
```
it's correct :+1:

exekias commented on metricbeat/docs/metricbeat-options.asciidoc:
```diff
@@ -1,30 +1,85 @@
+[[enable-modules-config-file]]
+=== Enable module configs in the +{beatname_lc}.yml+ file
+
+// REVIEWERS: I think the following statement is true.
```
I would say yes, but want @tsg opinion here

dedemorton replied:
@tsg Is the following statement true? Sounds like we would really prefer for users to use modules.d, right?

tsg replied:
Yes, it's true, we should guide people to use the `modules.d` folder.

dedemorton commented:
@exekias I've removed `enabled: true` from the examples (and deleted the questions flagged for reviewers). Do you have any other comments? Just waiting to hear back from tsg and get your LGTM. Thanks!

exekias commented:
:+1: on my side
